docs/guides/crd_guide.md

```diff
@@ -0,0 +1,497 @@
+        retentionCopies: 7
+        backupStorageName: "my-s3-backup-storage"
+    pitr:  # Point-in-Time Recovery
+      enabled: true
+      backupStorageName: "my-s3-backup-storage"
+      uploadIntervalSec: 300  # 5 minutes
+```
+
+### Manual Backups and Restores
+
+In addition to scheduled backups, you can create manual backups and perform restores using the `DatabaseClusterBackup` and `DatabaseClusterRestore` CRDs.
+
+#### Creating a Manual Backup
+
+```yaml
+apiVersion: everest.percona.com/v1alpha1
+kind: DatabaseClusterBackup
+metadata:
+  labels:
+    clusterName: my-database-cluster
+  name: my-database-cluster-backup
+spec:
+  backupStorageName: my-s3-backup-storage
+  dbClusterName: my-database-cluster
+```
+
+Monitor the backup status:
+```bash
+kubectl get dbbackup manual-backup-2024-04-11 -o jsonpath='{.status}'
+```
+
+The backup will go through these states:
+- `Starting`: Initial backup preparation
+- `Running`: Backup in progress
+- `Succeeded`: Backup completed successfully
+- `Failed`: Backup failed
+
+#### Restoring from a Backup
+
+You can restore from a backup in two ways:
+
+1. **Restore from a DatabaseClusterBackup**:
+```yaml
+apiVersion: everest.percona.com/v1alpha1
+kind: DatabaseClusterRestore
+metadata:
+  name: restore-from-backup
+spec:
+  dbClusterName: my-database-cluster
+  dataSource:
+    dbClusterBackupName: my-database-cluster-backup
+```
+
+2. **Restore to a New Database Cluster**:
+
+To restore a backup to a new database cluster, create a new `DatabaseCluster` with the `dataSource` field that references the backup:
+
+```yaml
+apiVersion: everest.percona.com/v1alpha1
+kind: DatabaseCluster
+metadata:
+  name: restored-database
+  namespace: default
+spec:
+  # .. hidden
+  dataSource:
+    dbClusterBackupName: backup-name-here  # Name of the backup to restore from
+```
+
+Important considerations when restoring to a new cluster:
+- The new cluster must use the same database engine type and version as the backup
+- The new cluster's storage size should be at least as large as the original cluster
+- The new cluster will be created in the same namespace as the backup
+- Other configurations (like proxy settings, monitoring, etc.) may be different from the original cluster
+
+#### Point-in-Time Recovery (PITR)
+
+You can perform point-in-time recovery to restore your database to a specific moment:
+
+```yaml
+apiVersion: everest.percona.com/v1alpha1
+kind: DatabaseClusterRestore
+metadata:
+  name: pitr-restore
+spec:
+  dbClusterName: my-database
+  dataSource:
+    dbClusterBackupName: base-backup
+    pitr:
+      type: date
+      date: "2024-04-11T15:30:00Z"  # UTC timestamp
+```
+
+Or restore to the latest available point:
+```yaml
+apiVersion: everest.percona.com/v1alpha1
+kind: DatabaseClusterRestore
+metadata:
+  name: latest-pitr
+spec:
+  dbClusterName: my-database
+  dataSource:
+    dbClusterBackupName: base-backup
+    pitr:
+      type: latest
+```
```

Comment:
although the CRs allow the `latest` restoration type I'm not sure if our QA ever touched that functionality and we never officially supported it. Should we still mention it in this guide?

Reply:
Let's have some third opinion here then, I'm not sure what if we should remove it or not... @recharte wdyt?

Reply:
> we never officially supported it

technically we also never officially supported using CRDs but here we are 😂 (jokes aside, yeah, lets see what Diogo has to say)

Reply:
since Diogo is on vacation, I'm removing this example so we can resolve and merge.. We can add back later if needed